test/selenium-tests/index.po.js

```diff
@@ -0,0 +1,107 @@
+import webdriver from 'selenium-webdriver'
+const {By} = webdriver
+
+export default class Index{
+    constructor(driver, url){
+        this.driver = driver
+        this.url = url
+    }
+
+    gotoIndex(){
+        return this.driver.navigate().to(this.url)
```

Comment:
add semicolon at the end of each statement

Reply:
updated